tests/dummy/app/components/dummy-status/dummy-status.js

```diff
@@ -0,0 +1,11 @@
+import Component from "@glimmer/component";
+
+export default class DummyStatusComponent extends Component {
+  get stati() {
```

Comment:
It seems that in english it's called `statuses` (https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status)
```suggestion
  get statuses() {
```

Reply:
Ah sorry I just realized this is only testing code. Feel free to leave it.

Reply:
I renamed it. Even if it is testing code it should use correct language.